10_files_lab/fileslab.txt

```diff
@@ -8,30 +8,53 @@ Online: https://www.tocode.co.il/bundles/unix1/lessons/fileslab
   - .highscore
 
 Write here the commands you used:
+cd
+mkdir lab
+cd lab
+pico a.out hero.c enemy.c game.c main.c human.h monster.h .highscore
+**Add some words to each file and save it
 
 2. "cd" into lab folder and inside create the following subdirectories:
   - Music
   - Misc
   - Drivers
 
 Write here the commands you used:
+I`m already in the folder, so:
+mkdir Music Misc Drivers
 
 3. Copy the file "/etc/passwd" to each subdirectory created in (2).
 
 Write here the commands you used:
+cp /etc/passwd /home/ym/lab/Music
+cp /etc/passwd /home/ym/lab/Misc
+cp /etc/passwd /home/ym/lab/Drivers
```

Comment:
that's correct,  but there is a nice shortcut for your 
home path called "~"  ->``` cp /etc/passwd ~/lab/Music```

or you can use relative paths like you did below :)